rueidiscompat/command.go

```diff
@@ -1959,6 +1959,7 @@ type SetArgs struct {
 
 type BitCount struct {
 	Start, End int64
+	Unit       *string // Stores BIT or BYTE
```

Comment:
Hi @SoulPancake,

Could we keep using `string` instead of `*string`? We'd like to make the interface as close to go-redis'.